docs/source/firstDraft.py

```diff
@@ -1,17 +1,18 @@
-from qgis.core import QgsProject
+def add_join_DruhPozemku(self, parcely_layer):
```

Comment:
Funkce by mohla byt pripraveno pro obecnejsi pouziti (zpusob vyuziti pozemku a dalsi ciselniky poskytovane CUZK).

```suggestion
def add_join_csv_file(self, parcely_layer, field_name, csv_name, csv_field="Kod", csv_subset=["NAZEV"]):
```

Zbytek kodu upravte na zakladne pridanych argumentu, to jiz necham na Vas :-)